install

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Reset
+Color_Off='\033[0m' # Text Reset
+
+# Regular Colors
+Red='\033[0;31m'   # Red
+Green='\033[0;32m' # Green
+Dim='\033[0;2m'    # White
+
+# Bold
+Bold_Green='\033[1;32m' # Bold Green
+Bold_White='\033[1m'    # Bold White
+
+error() {
+    echo -e "${Red}error${Color_Off}:" "$@" >&2
+    exit 1
+}
+
+info() {
+    echo -e "${Dim}$@ ${Color_Off}"
+}
+
+info_bold() {
+    echo -e "${Bold_White}$@ ${Color_Off}"
+}
+
+success() {
+    echo -e "${Green}$@ ${Color_Off}"
+}
+
+tildify() {
+    if [[ $1 = $HOME/* ]]; then
+        local replacement=\~/
+
+        echo "${1/$HOME\//$replacement}"
+    else
+        echo "$1"
+    fi
+}
+
+case $(uname -ms) in
+'Darwin x86_64')
+    target=x86_64-apple-darwin
+    ;;
+'Darwin arm64')
+    target=aarch64-apple-darwin
+    ;;
+'Linux x86_64' | *)
+    target=x86_64-unknown-linux-musl
+    ;;
+esac
+
+today=$(date +"%Y-%m-%d")
+dune_uri="https://download.dune.ci.dev/$today/$target/dune"
+
+install_dir=$HOME/.dune
+bin_dir=$install_dir/bin
+exe=$bin_dir/dune
+tilde_bin_dir=$(tildify $bin_dir)
+
+if [[ ! -d $bin_dir ]]; then
+    mkdir -p "$bin_dir" ||
+        error "Failed to create install directory \"$bin_dir\""
+fi
+
+curl --fail --location --progress-bar --output "$exe" "$dune_uri" ||
+    error "Failed to download dune from \"$dune_uri\""
+
+chmod +x "$exe" ||
+    error 'Failed to set permissions on dune executable'
+
+
+
+success "dune was installed successfully to $Bold_Green$(tildify "$exe")"
+
+echo
+
+case $(basename "$SHELL") in
+fish)
+    # Install completions, but we don't care if it fails
+    IS_dune_AUTO_UPDATE=true SHELL=fish $exe completions &>/dev/null || :
```

Comment:
I may be mistaken, but I don't thunk we have completion, no?

Reply:
Ah yes, remnants from the original bun.sh install script :D will remove.

Reply:
__todo (blocking)__: remove the completion part to avoid script error.